server/models/proposal.js

```diff
@@ -255,9 +266,12 @@ Proposal.schema = Joi.object({
 
 Proposal.routes = Hoek.applyToDefaults(AnchorModel.routes, {
   create: {
+    scope: ["coordinator", "root"],
     payload: Joi.object({
       userId: Joi.string().required(),
       fileName: Joi.string().required(),
+      groupId: Joi.string().optional(),
+      parentId: Joi.string().optional(),
```

Comment:
for groupId and parentId allow null and empty string for a more general cases. Optional addresses the case when the keys (groupId and parentId) don't exist in the payload object but if they exist and they have either a null value or are empty, it would return an error if we don't explicitly allow null and empty string.